.github/workflows/bump_submodules.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Submodule
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,4'  # Run at 00:00 on Sunday and Thursday
```

Comment:
Could also set this to run on workflow_dispatch, maybe instead of a cron. Not sure we want this spamming the repo with PRs